examples/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -100,24 +100,23 @@ class _ChipsTile extends StatelessWidget {
       }).toList()));
     } else {
       final TextStyle textStyle = Theme.of(context).textTheme.caption.copyWith(fontStyle: FontStyle.italic);
-      cardChildren.add(
-        Semantics(
-          container: true,
-          child: Container(
-            alignment: Alignment.center,
-            constraints: const BoxConstraints(minWidth: 48.0, minHeight: 48.0),
-            padding: const EdgeInsets.all(8.0),
-            child: Text('None', style: textStyle),
-          ),
-        ));
+      cardChildren.add(Semantics(
```

Comment:
The original formatting looks better to me. Why not leave the positional parameter on its own line if the developer put it there?